dist/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta http-equiv="X-UA-Compatible" content="IE=edge" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<link rel="icon" type="image/x-icon" href="f24f5b19881ff3f7ab93.ico" />
+		<title>To-Do List</title>
+	<link rel="icon" href="favicon.ico"><script defer src="main.js"></script></head>
```

Comment:
[OPTIONAL]
I notice you have two favicons linked to your page.
May you consider using only one icon.